.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
Also take a look at versioneye. It can track outdated dependencies

Sent from my iPhone

> On 13 jun. 2015, at 22:55, Mike van Riel <notifications@github.com> wrote:
> 
> In .gitignore:
> 
> > @@ -1,3 +1,2 @@
> >  .idea
> > -composer.lock
> Agreed; there is no policy currently but as we are typing I think we should set up a developer page and a few crons that check this for all repositories in the phpDocumentor organisation
> 
> —
> Reply to this email directly or view it on GitHub.
>